src/test/java/org/mockbukkit/mockbukkit/entity/ThrownPotionMockTest.java

```diff
@@ -104,7 +104,7 @@ void getPotionMeta_GivenValidPotion(Material input)
 
 		@NotNull PotionMeta actual = potion.getPotionMeta();
 		assertNotNull(actual);
-		assertSame(actual, potion.getPotionMeta());
+		assertEquals(actual, potion.getPotionMeta());
```

Comment:
If these two shouldn't be the same, that should also be checked with `assertNotSame`